tests/nav-spine_in-spine-no-list-style/OPS/content_001.xhtml

```diff
@@ -0,0 +1,8 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>TOC items ignore navigation list styling</title>
+  </head>
+  <body>
+    <p>Test passes if you can access the two links in the navigation document, which is in the spine.</p>
```

Comment:
I don't think this is the correct description.

Reply:
I have taken over your text in the package document, with a slight change.